src/app/core/auth/pages/api-keys/api-keys.page.ts

```
@@ -78,47 +78,69 @@ export class ApiKeysPage implements OnInit {
+  this.successMsg.set('');
+  this.isLoading.set(true);
+  try {
+    const result = await firstValueFrom(this.auth.createApiKey({ name }));
+
+    // 2. حفظ الـ rawKey في الـ Signal وفي الـ localStorage مع بعض
+    if (result.rawKey && result.key?.id) {
+      this.localRawKeys.update(prev => {
+        const updated = { ...prev, [result.key.id]: result.rawKey };
+        try {
+          localStorage.setItem('app_saved_api_keys', JSON.stringify(updated));
+        } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)